Drafts/1NNN-VLP.md

```diff
@@ -0,0 +1,65 @@
+# Aliases of member functions in template arguments
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | Victor Porton porton@narod.ru                                   |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | Draft                                                           |
+
+## Abstract
+
+Member functions should be passeable as alias template arguments.
+
+## Contents
+* [Rationale](#rationale)
+* [Description](#description)
+* [Copyright & License](#copyright--license)
+* [Reviews](#reviews)
+
+## Rationale
+
+"Regular" aliases can be assigned to a member function like this:
+
+```d
+struct S {
+  void f() { }
+}
+S s;
+alias f = s.f;
```

Comment:
I'm not sure how this is used, but it seems to be the same as `alias f = S.f;`. This example should include use of `f`. DIPs should make their whole case clearly.